docs/cni/kubernetes/AWS.md

```diff
@@ -259,63 +259,24 @@ Verify that you can access the Kubernetes API.  The following command should ret
 ```
 ./kubectl get nodes
 ```
+### 4.2 Configure Outbound NAT
 
-### 4.2 Deploying SkyDNS
-You now have a basic Kubernetes cluster deployed using Calico networking.  Most Kubernetes deployments use SkyDNS for Kubernetes service discovery.  The following steps configure the SkyDNS service.
+To enable connectivity to the internet for our Pods, we'll use `calicoctl`:
 
-Deploy the SkyDNS application using the provided Kubernetes manifest.
 ```
-./kubectl create -f manifests/skydns.yaml
-```
-
-Check that the DNS pod is running. It may take up to two minutes for the pod to start, after which the following command should show the `kube-dns-v9-xxxx` pod in `Running` state.
-```
-./kubectl get pods --namespace=kube-system
-```
-> Note: The kube-dns-v9 pod is deployed in the `kube-system` namespace.  As such, we we must include the `--namespace=kube-system` option when using kubectl.
-
->The output of the above command should resemble the following table.  Note the `Running` status:
-```
-NAMESPACE     NAME                READY     STATUS    RESTARTS   AGE
-kube-system   kube-dns-v9-3o2rw   4/4       Running   0          2m
-```
-
-### 4.3 Deploying the guestbook application
-You're now ready to deploy applications on your Cluster.  The following steps describe how to deploy the Kubernetes [guestbook application][guestbook].
-
-Create the guestbook application pods and services using the provided manifest.
-```
-./kubectl create -f manifests/guestbook.yaml
-```
-
-Check that the redis-master, redis-slave, and frontend pods are running correctly.  After a few minutes, the following command should show all pods in `Running` state.
-```
-./kubectl get pods
-```
-> Note: The guestbook demo relies on a number of docker images which may take up to 5 minutes to download.
-
-The guestbook application uses a NodePort service to expose the frontend outside of the cluster.  You'll need to allow this port outside of the cluster with a firewall-rule.
-```
-aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID \
-  --protocol tcp --port 30001 --cidr 0.0.0.0/0
+# Log into the master instance.
+ssh -i ~/mykey.pem core@$MASTER_DNS
 
+# Enable outgoing NAT on the Calico pool.
+docker run --rm --net=host calico/ctl pool add 192.168.0.0/16 --nat-outgoing
```

Comment:
Can you please confirm that we share the Kubernetes etcd cluster (on default port) on AWS?